controller.rst

```diff
@@ -134,7 +134,7 @@ and ``redirect()`` methods::
     use Symfony\Component\HttpFoundation\RedirectResponse;
 
     // ...
-    public function index()
+    public function index(): Response
```

Comment:
It seems to be common practice to add `use` statements to the code examples if a class is referenced. However, there is no such statement for `Response`. It's also missing in other examples where you've added the return type.

Reply:
Good catch. We don't always add all use statements (only the relevant use statements, or the first occurence of that class on the page). I've rechecked the examples with `Response` return typehint and added a few use statements that I think where missing.